fetch.php

```diff
@@ -0,0 +1,102 @@
+<?php
+ include('include/init.php');
+  if(isset($_POST["id"])){
+    $posts = displayCategory($_POST['id']); //All product posts from specific category
+    $output = '';
+    $customOutput = '';
+    // $customOutput = '';
+    if($_POST['id'] == 0) {
+      if(verifyUser()) {
+        $savedPosts = getSavedPosts($_SESSION['userId']);
+
+  //This case is only for displaying Saved Items
+      foreach($savedPosts as $saved) {
+        $prodId = $saved["productId"];
+        $product = getProduct($prodId);
+        $title = $product["postingTitle"];
+        $price = $product["price"];
+        $location = $product["location"];
+        $image = $product["image"];
+        
+    $customOutput .= '
+    <div class="col">
+      <div class="card shadow-sm">
+      <img class="bd-placeholder-img card-img-top" width="100%" height="225" src="'.$image.'" role="img" aria-label="Placeholder: Thumbnail" preserveAspectRatio="xMidYMid slice" focusable="false"><title>Placeholder</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">Thumbnail</text></svg>
+      <div class="card-body">
+        <p class="card-text">'.$title.'</p>
+        <p class="card-text">'.$price.'</p>
+        <p class="card-text">'.$location.'</p>
+        <div class="d-flex justify-content-between align-items-center">
+        <div class="btn-group">
+        <a href="description.php?productId='.$prodId.'"> <button type="button" class="btn btn-sm btn-outline-secondary">View Item</button> </a>
+          <button type="button" class="btn btn-sm btn-outline-secondary">Remove Item</button>
+        </div>
+        <small class="text-body-secondary">WashUList</small>
+      </div>
+    </div>
+  </div>
+  </div>
```

Comment:
I think I may have mentioned this before, but I think it would be better to return the objects as an array like (Title=>, Price=>, Location) -- then all of the layout code would be on the main page, and the ajax endpoint just fetches data.